docs/source/getting_started/active_learning.rst

```diff
@@ -179,13 +179,33 @@ Image Classification
 ^^^^^^^^^^^^^^^^^^^^^
 Use this scorer when working on a classification problem (binary or multiclass).
 
+Currently, the following scorers are available:
+
+- **prediction-margin** uses the margin between 1.0 and the highest confidence 
```

Comment:
While this description is correct in term of what we implemented, it is incorrect in terms of what is commonly understood, see: https://towardsdatascience.com/uncertainty-sampling-cheatsheet-ec57bc067c0b

Reply:
Yes, let's move this to another issue. We should provide more information about the scorers we implemented. For now the documentation should match the code.